### DISCUSSION TITLE
What is the difference between sniffnet and little snitch?

### DISCUSSION THREAD

**Discussion by specter78** [GyulyVGC/sniffnet#317] [Q&A]
What the title asks

**Comment by GyulyVGC** [comment]
I've never used little snitch, but from what I can see it works as a kind of firewall, while Sniffnet is more driven towards monitoring.
With Sniffnet you cannot black-list hosts/addresses or block a specific source of traffic.

**Comment by iulko** [comment]
In a post on reddit i read that you want sniffnet to become close to what Glasswire is, are you thinking of expanding to app blocking/firewall like Glasswire? I think there is no good alternative to Glasswire, and that would be like a thing to make this project go mainstream.

What is the best place to contact you directly?

**Comment by GyulyVGC** [comment]
First of all, thanks for your the interest in Sniffnet!

> are you thinking of expanding to app blocking/firewall like Glasswire? 

It's not currently programmed, but it could be something I can add to the next major features milestone.
See the [project roadmap](https://github.com/GyulyVGC/sniffnet/blob/main/ROADMAP.md) for more info.

> What is the best place to contact you directly?

Via e-mail or direct message on LinkedIN.
You can find my social links on https://github.com/GyulyVGC/GyulyVGC
